src/ui/components/atoms/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Tooltip } from './Tooltip';
+
+// Configuración del Storybook
+const meta: Meta<typeof Tooltip> = {
+  title: 'ui/components/atoms/Tooltip',
+  component: Tooltip,
+  argTypes: {
+    text: {
+      control: 'text',
+      description: 'Texto que se mostrará al usuario para interactuar.',
```

Comment:
Please change the language of your description test to English

Reply:
Description changes successfully completed